Lib/test/test_dict.py

```diff
@@ -1594,6 +1594,24 @@ def make_pairs():
                 self.assertEqual(d.get(key3_3), 44)
                 self.assertGreaterEqual(eq_count, 1)
 
+    def test_store_attr_with_hint(self):
+        # gh-133441: Regression test for STORE_ATTR_WITH_HINT bytecode
```

Comment:
This should be in `test_opcache` not `test_dict` IMO.

Reply:
I moved the test to test_opcache, but I'm not sure if TestInstanceDict is a good home for such test.